ui/src/main/webapp/src/app/services/application-group.service.ts

```diff
@@ -8,6 +8,9 @@ import {Observable} from "rxjs";
 
 @Injectable()
 export class ApplicationGroupService extends AbstractService {
+    @Output()
+    applicationGroupLoaded = new EventEmitter<ApplicationGroup>();
```

Comment:
I think `@Output()` decorator should be used only in components. IMHO it doesn't have any special meaning in services. Also `EventEmitter` is supposed to used only by directives and components. It should not be used in services. You can replace it by `Subject` and `emit()` method with `next()`. The remaining API should be the same. 
I recommend having 2 variables - one with `Subject` and one with `Observable` and publicly expose only the `Observable` one. To get Subject from observable, use `asObservable()` method.

See http://stackoverflow.com/questions/36076700/what-is-the-proper-use-of-an-eventemitter